source/Coop.Core/CoopartiveMultiplayerExperience.cs

```diff
@@ -21,11 +24,37 @@ public CoopartiveMultiplayerExperience()
         {
             harmony.PatchAll(typeof(GameInterface.GameInterface).Assembly);
             SurrogateCollection.AssignSurrogates();
+
+            MessageBroker.Instance.Subscribe<ConnectWithIP>(Handle);
+            MessageBroker.Instance.Subscribe<HostSave>(Handle);
         }
 
         ~CoopartiveMultiplayerExperience()
         {
             harmony.UnpatchAll(HarmonyId);
+
+            MessageBroker.Instance.Unsubscribe<ConnectWithIP>(Handle);
+            MessageBroker.Instance.Unsubscribe<HostSave>(Handle);
+        }
+
+        private void Handle(MessagePayload<ConnectWithIP> obj)
+        {
+            var connectMessage = obj.What;
+
+            var config = new NetworkConfiguration()
+            {
+                Address = connectMessage.Address.ToString(),
```

Comment:
Address in debug is read only, fails to build

Reply:
Fixed